src/js/main.js

```diff
@@ -9,6 +9,29 @@ const default_prefs = {
   'shabadToggles': []
 };
 
+function getPrefs() {
+  $.each(default_prefs, function(key, defaults) {
+    prefs[key] = window.localStorage[key] ? JSON.parse(window.localStorage[key]) : defaults;
+  });
+}
+function setPref(key, val) {
+  window.localStorage[key] = JSON.stringify(val);
+}
+function checkboxPref(e, key, option) {
+  if (e.target.classList.contains('active')) {
+    if (prefs[key].indexOf(option) < 0) {
+      prefs[key].push(option);
+    }
+  } else {
+    const index = prefs[key].indexOf(option);
+    prefs[key].splice(index, 1);
+  }
+  setPref(key, prefs[key]);
+}
+
+// IE 10 doesn't support toggle class
+const toggleClass = ($el, cl) => $el.classList[$el.classList.contains(cl) ? 'remove' : 'add'](cl);
```

Comment:
This is fine for now, but we are only supporting back to IE11. Any system that supports IE10 also supports IE11.

Reply:
On further review, it seems that this is used extensively. I would rather we kept `el.classList.toggle`